config/routes.rb

```diff
@@ -1,11 +1,14 @@
 Rails.application.routes.draw do
+  # Defines the root path route ("/")
+  root "photos#index"
```

Comment:
I also think it's better to write routes in the order you actually open pages in your app. Great job!